### DISCUSSION TITLE
Textblock shows nothing with timespan string format

### DISCUSSION THREAD

**Discussion by THE-KONDRAT** [AvaloniaUI/Avalonia#15679] [Q&A]
### Describe the bug

I want to show timespan in some format but i see nothing when i add string format property in binding.
I tried [that](https://github.com/AvaloniaUI/Avalonia/issues/9240#issuecomment-2082458277) way but i still see nothing.

### To Reproduce

axaml:
```axaml
<TextBlock
    Grid.Row="0"
    Grid.Column="1"
    HorizontalAlignment="Right"
    VerticalAlignment="Center"
    Text="{Binding RenderingElapsed, StringFormat='hh:mm:ss'}" />
```
and property in a viewmodel
```csharp
[ObservableProperty]
private TimeSpan renderingElapsed = new(12, 12, 34, 56);
```

### Expected behavior

Timespan shows like that: `12:34:56`

### Avalonia version

11.1.0-beta2, 11.0.10

### OS

Windows

### Additional context

![obraz](https://github.com/AvaloniaUI/Avalonia/assets/87300126/bae8c417-96fa-4e69-be11-3cac4ae32dfc)
![obraz](https://github.com/AvaloniaUI/Avalonia/assets/87300126/f48be574-74ab-4d4a-b452-f57b315b4f84)

**Comment by zii-dmg** [comment]
TimeSpan ToString rules is not same as DateTime. Use escape for colons `Text="{Binding Time, StringFormat='hh\\:mm\\:ss'}"`

**Reply by THE-KONDRAT** [reply]
Ok. It works. I tried with backslash before but i forgot about double backslash.
